frontend/src/hooks/useIsMobile/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './useIsMobile';
```

Comment:
Look, we don't make a separate folder for the hook, it should be src/hooks/file, and everything that doesn't render anything we do through a named export